test/e2e/network/netpol/probe.go

```diff
@@ -90,27 +88,22 @@ func probeWorker(k8s *kubeManager, jobs <-chan *ProbeJob, results chan<- *ProbeJ
 	defer ginkgo.GinkgoRecover()
 	for job := range jobs {
 		podFrom := job.PodFrom
-		containerFrom, err := podFrom.FindContainer(int32(job.FromPort), job.Protocol)
```

Comment:
Is this the only place `FindContainer` was used ?  If so, probably best to remove it.